src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PerformTouchAction.java

```
@@ -0,0 +1,360 @@
+		System.out.println("appWebviewWidth: " + appWebviewWidth); //360
+		System.out.println("=======================================");
+		System.out.println("loadedPageHeight: " + loadedPageHeight); //2484
+		System.out.println("loadedPageWidth: " + loadedPageWidth); //360
+		System.out.println("=======================================");
+	}
+	
+	/**
+	 * 
+	 * @param direction Use public const DIRECTION from that class
+	 * @param pixelPath	From 0 to half of screen X or Y, if path will be too high it will be set to edge of app
+	 * @param duration In milliseconds
+	 * @param waitAfter In milliseconds, Reccomend 2*duration
+	 */
+	public void swipeFromCenterToDirection(String direction, int pixelPath, int duration, int waitAfter){
+		int centerX = appNativeWidth / 2;
+		int centerY = (appNativeHeight / 2) + taskbarNativeHeight;
+		int path = 0;
+		if (mobileDriver.getContext() != "NATIVE_APP") {
+			mobileDriver.context("NATIVE_APP");
+		}
+		switch (direction) {
+			case "left": 
+				if (pixelPath < centerX) {
+					path = pixelPath;
+				} else {
+					path = centerX - 2;
+				}
+				try {
+					mobileDriver.swipe(centerX, centerY, centerX - path, centerY, duration);
+				} catch (Exception e) {}
+				break;
+			case "right": 
+				if (pixelPath < centerX) {
+					path = pixelPath;
+				} else {
+					path = centerX - 2;
+				}
+				try {
+					mobileDriver.swipe(centerX, centerY, centerX + path, centerY, duration);	
+				} catch (Exception e) {}
+				break;
+			case "up": 
+				if (pixelPath < centerY) {
+					path = pixelPath;
+				} else {
+					path = centerY - 2;
+				}
+				try {
+					mobileDriver.swipe(centerX, centerY, centerX, centerY - path, duration);				
+				} catch (Exception e) {}
+				break;
+			case "down": 
+				if (pixelPath < centerY) {
+					path = pixelPath;
+				} else {
+					path = centerY - 2;
+				}
+				try {						
+					mobileDriver.swipe(centerX, centerY, centerX, centerY + path, duration);	
+				} catch (Exception e) {}
```

Please use a more specific Exception that the try block is catching.
Also, please either throw the exception or log the exception in the catch block.